README.md

```diff
@@ -1,5 +1,34 @@
 # open-source-project
 
+# Number guessing game
```

Comment:
can we move this complete section below the group members section

Reply:
Moved